src/services/item/plugins/publication/README.md

```diff
@@ -0,0 +1,101 @@
+  - Currently only folders are allowed to be published.
+- **Unpublished**: The item is not yet published in the Library.
+  - This is the default state for new items.
+  - Validation is required before publication.
+- **Outdated**: The item is currently published but has been updated, necessitating re-validation.
+  - Currently, outdated items remain in the Library without removal, but this behavior might change to include a warning.
+- **Invalid**: The item contains invalid content and has been rejected.
+- **Pending**: The item's validation is in progress.
+  - Successful validation transitions the item to Published.
+  - Invalid validation transitions the item to Invalid.
+- **NotPublic**: The item is published but has private visibility.
+- **Published**: The item is successfully published in the Library.
+  - Updating the item changes the status to Outdated.
+  - Unpublishing changes it to ReadyToPublish.
+- **ReadyToPublish**: The item is unpublished but has passed validation.
+  - If not updated, it transitions to Published.
+  - Updating the item reverts the status to Unpublished.
+
+```mermaid
+stateDiagram-v2
+    state if_published_child <<choice>>
+    state if_item_not_allowed <<choice>>
+    [*] --> IsItemAllowed
+    IsItemAllowed --> if_item_not_allowed
+    if_item_not_allowed --> ItemTypeNotAllowed: No
+    if_item_not_allowed --> IsParentPublished: Yes
+    IsParentPublished --> if_published_child
+    if_published_child --> PublishedChildren: Yes
+    if_published_child --> Unpublished: No
+    Unpublished --> Pending : validate
+    Pending --> Published : valid
+    Pending --> Invalid : invalid
+    ReadyToPublish --> Published: publish
+    ReadyToPublish --> Unpublished: update
+    Published --> Outdated : update
+    Published --> ReadyToPublish : unpublish
+    Published --> NotPublic: private
+    Published --> [*]
+    Outdated --> Pending : revalidate
+    Invalid --> [*]
+    NotPublic --> Published: public
+    PublishedChildren --> [*]
+    ItemTypeNotAllowed --> [*]
+```
+
+- **Endpoint:** `GET /items/publications/:itemId`
+  - **Description:** Gets the status of the item's publication.
+  - **Permissions:** Only **admin** members of the item can access this route.
+  - **Responses:**
+    - `200 OK`: The item exists.
+    - `401 Not Authorized`: If the user is not authenticated.
+    - `403 Forbidden`: If the user is not administrator of the item.
+    - `404 Not Found`: If the item is not found.
+
+## Validation service
+
+The `ValidationService` allows to validate an item or to retrieve an item's validation. This service is crucial to ensure that an item is validated before publishing it in the Graasp's Library.
+
+- **Endpoint:** `POST /items/:itemId/validations`
+  - **Description:** Validates the item's content.
+  - **Permissions:** Only **admin** members of the item can access this route.
+  - **Behavior:**
+    - Validation is asynchronous with websocket feedback.
```

Comment:
```suggestion
    - Validation is asynchronous. Websocket feedbacks are sent to the frontend ....
```
complete with websocket messages, doesn't have to be precise, but would be helpful to know when.